tests/manage/pv_services/pv_encryption/test_pvc_with_tc_overrie_vn.py

```diff
@@ -0,0 +1,178 @@
+import logging
+import pytest
+
+from ocs_ci.ocs import constants
+from ocs_ci.framework import config
+from ocs_ci.framework.testlib import (
+    skipif_ocs_version,
+    ManageTest,
+    tier1,
+    bugzilla,
+)
+
+from ocs_ci.helpers.helpers import (
+    create_unique_resource_name,
+    create_pods
+)
+
+from ocs_ci.ocs.exceptions import ResourceNotFoundError
+from ocs_ci.utility import kms
+
+log = logging.getLogger(__name__)
+
+kmsprovider = constants.VAULT_KMS_PROVIDER
+argnames = ["kv_version", "kms_provider", "use_vault_namespace"]
+if config.ENV_DATA.get("vault_hcp"):
+    argvalues = [
+        pytest.param(
+            "v1", kmsprovider, True, marks=pytest.mark.polarion_id("OCS-4639")
+        ),
+        pytest.param(
+            "v2", kmsprovider, True, marks=pytest.mark.polarion_id("OCS-4641")
+        ),
+    ]
+else:
+    argvalues = [
+        pytest.param(
+            "v1", kmsprovider, False, marks=pytest.mark.polarion_id("OCS-4638")
+        ),
+        pytest.param(
+            "v2", kmsprovider, False, marks=pytest.mark.polarion_id("OCS-4640")
+        ),
+    ]
+
+
+@pytest.mark.parametrize(
+    argnames=argnames,
+    argvalues=argvalues,
+)
+@tier1
+@skipif_ocs_version("<4.8")
+@bugzilla("2050056")
+class TestEncryptedRbdTenantConfigmapOverride(ManageTest):
+    """
+    Tests to check Tenant configmap override vault namespace or not
+    1. Create a new namespace in OCP and create the ceph-csi-kms-token secret
+    2. Create an encryption enabled storageclass using vault namespaces
+    3. Create a configmap in the tenant namespace to override the vault namespace
+    4. Update the ceph-csi-kms-token secret in the tenant namespace to provide access to the new namespace and backend
+       path
+    5. Create a new PVC
+    6. Verify that the PVC is bound and the key is created in the specified path in Vault
+    7. Mounting the PVC in an app pod and running IO
+
+    """
+
+    @pytest.fixture(autouse=True)
+    def setup(
+        self,
+        kv_version,
+        kms_provider,
+        use_vault_namespace,
+        pv_encryption_kms_setup_factory,
+        project_factory,
+        storageclass_factory,
+        pod_factory,
```

Comment:
This is not used and can be removed from here

Reply:
Removed